.gitignore

```diff
@@ -8,3 +8,5 @@ __pycache__/
 .DS_Store
 **/*.pyc
 **.pyc
+*.tar.gz
+.DS_Store
```

Comment:
`.DS_Store` is already the first entry. also, nit: normally that is the kind of file that would go in one's global gitignore